src/edu/stanford/nlp/parser/lexparser/LexicalizedParser.java

```diff
@@ -1501,20 +1505,31 @@ public static void main(String[] args) {
       // test parser on treebank
       EvaluateTreebank evaluator = new EvaluateTreebank(lp);
       evaluator.testOnTreebank(testTreebank);
-    } else if (argIndex >= args.length) {
-      // no more arguments, so we just parse our own test sentence
+    } else {
+      // Ignore further arguments
       PrintWriter pwOut = op.tlpParams.pw();
       PrintWriter pwErr = op.tlpParams.pw(System.err);
-      ParserQuery pq = lp.parserQuery();
-      if (pq.parse(op.tlpParams.defaultTestSentence())) {
-        lp.getTreePrint().printTree(pq.getBestParse(), pwOut);
-      } else {
-        pwErr.println("Error. Can't parse test sentence: " +
-                      op.tlpParams.defaultTestSentence());
-      }
-    } else {
-      // We parse filenames given by the remaining arguments
-      ParseFiles.parseFiles(args, argIndex, tokenized, tokenizerFactory, elementDelimiter, sentenceDelimiter, escaper, tagDelimiter, op, lp.getTreePrint(), lp);
+
+      try {
+          pwOut.println("<READY>");
+          BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+          String input;
+          while ((input = br.readLine()) != null) {
+              //System.out.println("The sentence is: " + input);
+              ParseFiles.parseFiles(input, tokenized, tokenizerFactory, elementDelimiter, sentenceDelimiter, escaper, tagDelimiter, op, lp.getTreePrint(), lp);
+              pwOut.println("<READY>");
+          }
+          //System.out.println("Bye!");
+      } catch(IOException io) {
+          io.printStackTrace();
+      }       
+      //ParserQuery pq = lp.parserQuery();
+      //if (pq.parse(op.tlpParams.defaultTestSentence())) {
+        //lp.getTreePrint().printTree(pq.getBestParse(), pwOut);
+      //} else {
+        //pwErr.println("Error. Can't parse test sentence: " +
+                      //op.tlpParams.defaultTestSentence());
+      //}
```

Comment:
If we don't need this any more, should it be deleted instead of left in but commented out?

Reply:
Yeah I can remove those comments.  Should I do a commit --amend and then a push?